bundle/src/main/java/com/jfrog/ide/eclipse/scan/GradleScanManager.java

```diff
@@ -57,39 +60,17 @@ public static boolean isApplicable(IProject project) {
 
 	@Override
 	void refreshDependencies(IProgressMonitor monitor) throws IOException {
-		this.monitor = monitor;
-		String rootProjectDir = project.getLocation().toPortableString();
-		if (project.getLocation().toFile().isDirectory()) {
-			rootProjectDir = project.getLocation().addTrailingSeparator().toPortableString();
-		}
-
-		String gradleFileNameFullPath = "/gradle/" + GRADLE_INIT_SCRIPT;
-		ClassLoader classLoader = GradleScanManager.class.getClassLoader();
-		// classLoader.getResourceAsStream(gradleFileNameFullPath) will work on all the
-		// OSes
-		try (InputStream res = classLoader.getResourceAsStream(gradleFileNameFullPath)) {
-			String gradleFile = createGradleFile(res);
-			if (StringUtils.isBlank(gradleFile)) {
-				getLog().warn("Gradle init script wasn't created.");
-				return;
-			}
-			generateDependenciesGraphAsJsonTask(rootProjectDir, gradleFile);
-			parseJsonResult();
-		}
 	}
 
 	@Override
-	void buildTree() {
-		DependenciesTree rootNode = new DependenciesTree(getProjectName());
-		GeneralInfo generalInfo = new GeneralInfo();
-		generalInfo.groupId(gradleArtifact.getGroupId()).artifactId(gradleArtifact.getArtifactId())
-				.version(gradleArtifact.getVersion());
-		rootNode.setGeneralInfo(generalInfo);
-		GradleArtifact[] dependencies = gradleArtifact.getDependencies();
-		if (ArrayUtils.isNotEmpty(dependencies)) {
-			populateDependenciesTree(rootNode, dependencies);
+	void buildTree() throws IOException {		
+		try {
+			setScanResults(gradleTreeBuilder.buildTree(getLog()));
+		} 
+		catch (IOException ex) {
+			Logger.getInstance().error("Could not scan project: " + getProjectName() + ". Reason is: " + ex.getMessage());
```

Comment:
we are not throwing the exception anymore, please validate no issues or unexpected bugs occurs